ProvisionSample/Program.cs

```diff
@@ -79,6 +79,7 @@ static void Main(string[] args)
                 Console.WriteLine("7. Update connection string info for an existing dataset");
                 Console.WriteLine("8. Retrieve a list of Datasets published to a workspace");
                 Console.WriteLine("9. Delete a published dataset from a workspace");
+                Console.WriteLine("0. Get import status of deployment");
```

Comment:
I think the phrasing of this new option could be confusing/misleading to others.  I don't think deployment makes sense in the context of Importing/Uploading a .pbix file. 

Can you please change the text to: `0. Get import status` ?